auth/api/iam/api.go

```diff
@@ -412,7 +428,53 @@ func (r Wrapper) GetRequestJWT(ctx context.Context, request GetRequestJWTRequest
 // Extension of OpenID 4 Verifiable Presentations (OpenID4VP) on
 // RFC9101: The OAuth 2.0 Authorization Framework: JWT-Secured Authorization Request (JAR).
 func (r Wrapper) PostRequestJWT(ctx context.Context, request PostRequestJWTRequestObject) (PostRequestJWTResponseObject, error) {
```

Comment:
this sounds like something is POST-ed, but it handles a POST. Maybe `RequestJWTByGET` and `RequestJWTByPOST` or something? ("through" is more accurate but more stuff to write)

Reply:
it's only in the API, so it only hurts your eyeballs right now but not after you've merged the PR :-p